python/python/glide/async_commands/transaction.py

```diff
@@ -1966,6 +1967,46 @@ def zdiffstore(
             RequestType.ZDiffStore, [destination, str(len(keys))] + keys
         )
 
+    def bzmpop(
+        self: TTransaction,
+        keys: List[str],
+        modifier: ScoreFilter,
+        timeout: float,
+        count: Optional[int] = None,
+    ) -> TTransaction:
+        """
+        Blocks the connection until it pops and returns a member-score pair from the first non-empty sorted set, with
+        the given keys being checked in the order they are provided. The optional `count` argument can be used to
+        specify the number of elements to pop, and is set to 1 by default. The number of popped elements is the minimum
+        from the sorted set's cardinality and `count`.
+
+        BZMPOP is the blocking variant of ZMPOP.
+
+        BZMPOP is a client blocking command, see https://github.com/aws/glide-for-redis/wiki/General-Concepts#blocking-commands for more details and best practices.
+
+        See https://valkey.io/commands/bzmpop for more details.
+
+        Args:
+            keys (List[str]): The keys of the sorted set.
+            modifier (ScoreFilter): The element pop criteria - either ScoreFilter.MIN or ScoreFilter.MAX to pop
+                members with the lowest/highest scores accordingly.
+            timeout (float): The number of seconds to wait for a blocking operation to complete. A value of 0 will
+                block indefinitely.
+            count (Optional[int]): The number of elements to pop.
+
+        Command response:
+            Optional[List[Union[str, Dict[str, float]]]]: A two-element list containing the key name of the set from
+                which elements were popped, and a member-score dict. If no members could be popped and the timeout
+                expired, returns None.
+
+        Since: Redis version 7.0.0.
```

Comment:
Should be a line break after the header?

Reply:
Between the end of the documentation and the code you mean? The formatting surrounding this line matches the existing documentation, eg zrank_withscore contains this same line and also has an empty line above it and no empty line between the end of the documentation and the code